src/system-test/WebApi.MsBuild.SystemTest/openapi-v1-management.yaml

```diff
@@ -0,0 +1,47 @@
+openapi: 3.0.1
```

Comment:
This was generated by a tool or was created manually?

Reply:
By the tool. We have this as a baseline for comparison with `oasdiff`

Reply:
All right I really need to look on what this tool offers XD